AggregateMetrics/AggregateMetrics.Tests/AzureWebApp/AzureWebAppTest.cs

```diff
@@ -18,7 +18,7 @@ public void TestPerformanceCounterValuesAreCorrectlyRetrievedUsingFlexiblePerfor
             MetricTelemetry metric = new MetricTelemetry();
 
             metric.Name = performanceCounter;
-            metric.Value = CacheHelper.Instance.GetCounterValueHttp(performanceCounter);
+            metric.Value = CacheHelper.Instance.GetCounterValueFromHttpClient(performanceCounter);
```

Comment:
this should not be a part of product code. There should be a separate cache helper that is only used in tests